projects/addon-editor/components/edit-link/edit-link.template.html

```diff
@@ -1,20 +1,35 @@
 <ng-container *ngIf="edit; else view">
-    <label class="t-label">
-        URL
-        <span class="t-url">
-            {{ prefix }}
+    <label
+        *tuiLet="prefix === '#' as prefixIsHashMode"
```

Comment:
This can be a getter, no need for `tuiLet`

Reply:
fixed